extensions/typescript-language-features/src/features/tsconfig.ts

```diff
@@ -26,23 +26,30 @@ class TsconfigLinkProvider implements vscode.DocumentLinkProvider {
 		}
 
 		return [
-			this.getExendsLink(document, root),
+			this.getExtendsLink(document, root),
 			...this.getFilesLinks(document, root),
 			...this.getReferencesLinks(document, root)
 		].filter(x => !!x) as vscode.DocumentLink[];
 	}
 
-	private getExendsLink(document: vscode.TextDocument, root: jsonc.Node): vscode.DocumentLink | undefined {
+	private getExtendsLink(document: vscode.TextDocument, root: jsonc.Node): vscode.DocumentLink | undefined {
 		const extendsNode = jsonc.findNodeAtLocation(root, ['extends']);
 		if (!this.isPathValue(extendsNode)) {
 			return undefined;
 		}
 
+		if (extendsNode.value.startsWith('./') || extendsNode.value.startsWith('../')) {
```

Comment:
Not sure if TS supports windows style paths but to be safe I think this condition can just be `startsWith('.')`.